test/test-dbh.js

```diff
@@ -55,7 +97,7 @@ describe('Dbh', function () {
         });
         it('throws when given an unknown build tool', function () {
             assert.throws(() => {
-                let foo = dbh.getFilesWithNamingStrategy('foo');
+                let foo = dbh.getFilesWithNamingStrategy('foo'); // eslint-disable-line no-unused-vars, prefer-const
```

Comment:
The assignment raises the error, that's what we want when testing a block of code that should throw an exception.